src/reduced_order/min_max_scaler.cpp

```diff
@@ -0,0 +1,43 @@
+#include "min_max_scaler.h"
+
+namespace PHiLiP {
+namespace ProperOrthogonalDecomposition {
+
+MatrixXd MinMaxScaler::fit_transform(const MatrixXd &parameters){
+    min.resize(parameters.cols());
+    max.resize(parameters.cols());
+    MatrixXd parameters_scaled(parameters.rows(), parameters.cols());
+    for(int j = 0 ; j < parameters.cols() ; j++){
+        min(j) = parameters.col(j).minCoeff();
+        max(j) = parameters.col(j).maxCoeff();
+        for(int k = 0 ; k < parameters.rows() ; k++){
+            parameters_scaled(k, j) = (parameters(k, j) - min(j)) / (max(j) - min(j));
```

Comment:
I have added a check to warn the user if max(j) = min(j), but that would imply that there is no parameter range so that shouldn't really happen. I modeled the class after a similar Scipy class https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html 
It's sort of implied that you need to fit the scaler before you transform the data. So you can't really be calling transform without first calling fit_transform.